### PR TITLE
Changes to use markdown formating. issue #2

### PR DESCRIPTION
**Pull request by priyda**
changes to fix issue #28

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/Solidity/pull/48"></a>

### CODE REVIEW

File: src/Maths/AddArray.sol [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-        /// @dev Return the total.
-        total = sum;
-    }
-}
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title Array Addition.
+ * @author Anthony (fps)  (https://github.com/fps8k)
```
Comment (Panquesito7) [inline src/Maths/AddArray.sol:6]:
```suggestion
 * @author [Anthony (fps)](https://github.com/fps8k)
```

File: src/Maths/AliquotSum.sol [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-        }
-        return result;
-    }
-}
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title Get the absoule value of an integer.
+ * @author Perelyn (https://github.com/Perelyn-sama)
```
Comment (Panquesito7) [inline src/Maths/AliquotSum.sol:6]:
```suggestion
 * @author [Perelyn](https://github.com/Perelyn-sama)
```

File: src/Maths/Area.sol [hunk 3]
```diff
@@ -1,138 +1,138 @@
...
-    }
-}
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+/**
+ * @title Calculate the area of various shapes
+ * @author Perelyn (https://github.com/Perelyn-sama)
```
Comment (Panquesito7) [inline src/Maths/Area.sol:8]:
```suggestion
 * @author [Perelyn](https://github.com/Perelyn-sama)
```

File: src/Maths/AverageMean.sol [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
-
-        result = sum / _nums.length;
-    }
-}
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title find the mean value of an array of numbers
+ * @author Perelyn (https://github.com/Perelyn-sama)
```
Comment (Panquesito7) [inline src/Maths/AverageMean.sol:6]:
```suggestion
 * @author [Perelyn](https://github.com/Perelyn-sama)
```

File: src/Maths/MulArray.sol [hunk 5]
```diff
@@ -1,34 +1,34 @@
...
-        /// @dev Return the product.
-        product = _product;
-    }
-}
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title Array Multiplication.
+ * @author Anthony (fps)  (https://github.com/fps8k)
```
Comment (Panquesito7) [inline src/Maths/MulArray.sol:6]:
```suggestion
 * @author [Anthony (fps)](https://github.com/fps8k)
```

File: src/Maths/Percentage.sol [hunk 6]
```diff
@@ -1,128 +1,128 @@
...
-            percentage = int256(_percentage);
-        }
-    }
-}
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.0;
+
+/**
+ * @title Percentage Calculator Contract.
+ * @author Anthony (fps)  (https://github.com/fps8k)
```
Comment (Panquesito7) [inline src/Maths/Percentage.sol:6]:
```suggestion
 * @author [Anthony (fps)](https://github.com/fps8k)
```

File: src/Maths/armstrongNumber.sol [hunk 7]
```diff
@@ -1,43 +1,43 @@
...
-        return _n == sum;
-    }
-}
+pragma solidity ^0.8.0;
+
+/**
+ * @title This is a Solidity implementation to Armstrong number.
+ * @notice An Armstrong number is a number that is equal to the sum of
+ * cubes of its digits.
+ * @author Wenceslas Sanchez (https://github.com/wenceslas-sanchez)
```
Comment (Panquesito7) [inline src/Maths/armstrongNumber.sol:7]:
```suggestion
 * @author [Wenceslas Sanchez](https://github.com/wenceslas-sanchez)
```

File: src/Maths/factorial.sol [hunk 8]
```diff
@@ -1,37 +1,37 @@
...
-        }
-
-        return result;
-    }
-}
+pragma solidity ^0.8.0;
+
+/**
+ * @title This is a Solidity implementation to factorial.
+ * @author Wenceslas Sanchez (https://github.com/wenceslas-sanchez)
```
Comment (Panquesito7) [inline src/Maths/factorial.sol:5]:
```suggestion
 * @author [Wenceslas Sanchez](https://github.com/wenceslas-sanchez)
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
You might want to do the same for all the other files you modified. 🙂

**Comment by priyda** [comment]
> You might want to do the same for all the other files you modified. 🙂

On files I modified were not merged into main branch so didn't modify them. should I make another pull request?

**Comment by mkubdev** [comment]
Hi @Panquesito7, can we commit our suggestion?

**Comment by Panquesito7** [comment]
> > You might want to do the same for all the other files you modified. 🙂
> 
> On files I modified were not merged into main branch so didn't modify them. should I make another pull request?

Hmm, what do you mean exactly? 🤔

**Comment by priyda** [comment]
> > > You might want to do the same for all the other files you modified. 🙂
> > 
> > 
> > On files I modified were not merged into main branch so didn't modify them. should I make another pull request?
> 
> Hmm, what do you mean exactly? 🤔

please ignore, I made you confused  :) , I will commit suggestions , thank you :)

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🚀

**Pull request merged**